react/src/pages/SessionLauncherPage.tsx

```diff
@@ -1133,6 +1133,10 @@ const SessionLauncherPage = () => {
                           <>
                             <Descriptions.Item
                               label={t('session.launcher.StartUpCommand')}
+                              contentStyle={{
+                                overflow: 'scroll',
+                                width: '100%',
```

Comment:
This code makes padding in dark mode.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/3c2c64fe-4839-4a5d-bcce-e2fcc96b3931.png)

If you modify it like this,
```suggestion
                                overflow: 'auto',
```
you can remove unexpected padding :)
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/2a978736-b13c-4f58-86c2-78c3807cb708.png)

Reply:
Applied!